### DISCUSSION TITLE
Issue template

### DISCUSSION THREAD

**Discussion by NiceGuyIT** [Ylianst/MeshCentral#4389] [Ideas]
To reduce the confusion between bug reports vs feature requests vs help / support / general discussions, you can use issue templates. While you can read [GitHub's documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates), a good example of what's possible can be found in the [astro repo](https://github.com/withastro/astro/issues/new/choose) (click on Get started to see the template in action). One aspect that may be appealing is the [blank_issues_enabled](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) option which encourages users to use the template.

**Comment by si458** [comment]
brilliant suggestion!
ive created a PR to add unoffical channels for people and disable the blank issues - #4390
if you would like you can create a PR to create more issue templates which might help others
